trackomatic.js

```diff
@@ -114,17 +147,22 @@ function Trackomatic(tracker, config) {
   window.addEventListener('keydown', function(event) {
     // this is pretty fuzzy, but we want to filter out keys like space, up, down, etc
     // that a keyboard user might initially use in place of scrolling
-    var nonViewKey = true
-    if (event.metaKey || event.ctrlKey || event.altKey) nonViewKey = false
-    for (key in nonViewKeys) {
-    if (event.keyCode === nonViewKeys[key]) {
+    var nonViewKey = true;
+
+    if (event.metaKey || event.ctrlKey || event.altKey) {
       nonViewKey = false;
     }
+
+    for (key in nonViewKeys) {
+      if (event.keyCode === nonViewKeys[key]) {
+        nonViewKey = false;
+      }
     }
+
     if (nonViewKey) {
-    if (!firstInputRecorded) dataLayer.push({'fed-firstkeycode': event.keyCode})
-    var keyCode = event.keyCode;
-    recordInput('keyboard', keyCode);
+      if (!firstInputRecorded) dataLayer.push({ 'fed-firstkeycode': event.keyCode });
+      var keyCode = event.keyCode;
+      recordInput('keyboard', keyCode);
```

Comment:
this:

![screen shot 2015-09-10 at 3 39 06 pm](https://cloud.githubusercontent.com/assets/72114/9801726/17c20222-57d2-11e5-803b-7748ffc5a51b.png)